O/OpenSpiel/build_tarballs.jl

```diff
@@ -55,11 +55,12 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    BuildDependency(PackageSpec(name="libjulia_jll", version=julia_version)),
+    BuildDependency("libjulia_jll"),
     Dependency("libcxxwrap_julia_jll")
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
     preferred_gcc_version=v"9",
-    julia_compat="$(julia_version.major).$(julia_version.minor)")
+    julia_compat="1.6"
```

Comment:
Why this change?  Looks like `$(julia_version.major).$(julia_version.minor)` would still be correct, no?

Reply:
Yes, you're correct here.